libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -386,6 +392,17 @@ uint8_t AP_BattMonitor::capacity_remaining_pct(uint8_t instance) const
     }
 }
 
+/// time_remaining - returns remaining battery time
+bool AP_BattMonitor::time_remaining(int32_t &min, uint8_t instance) const
```

Comment:
This should be seconds.

And presumably unsigned?

Reply:
The type of time_remaining in the MAVLink2 BATTERY_STATUS message is int32_t. Would it be better to use this type?